src/registrar/templates/application_authorizing_official.html

```diff
@@ -12,8 +12,30 @@ <h2>Who is the authorizing official for your organization?</h2>
   <p>Your authorizing official is the person within your organization who can authorize your domain request. This is generally the highest ranking or highest elected official in your organization. Read more about <a href="{% url 'todo' %}">who can serve as an authorizing official</a>.
   </p>
 
-  <div class="ao-example">
-    {% include "includes/ao_example__city.html" %}
+  <div class="ao_example">
+    {% if is_federal %}
+      {% if federal_type == 'executive' %}
```

Comment:
For code organization, I might have favored putting this `if` block in an include and all of the example text inside of it over multiple smaller files. That's a matter of preference and this is fine.